heudiconv/parser.py

```diff
@@ -24,6 +25,7 @@
 
 _VCS_REGEX = r'%s\.(?:git|gitattributes|svn|bzr|hg)(?:%s|$)' % (op.sep, op.sep)
 
+_UNPACK_FORMATS = tuple(chain.from_iterable([x[1] for x in shutil.get_unpack_formats()]))
```

Comment:
sweet, just FYI although not even sure if preferable, but more "native python" way is
```suggestion
_UNPACK_FORMATS = tuple(sum((x[1] for x in shutil.get_unpack_formats()), []))
```
or just
```suggestion
_UNPACK_FORMATS = sum((x[1] for x in shutil.get_unpack_formats()), [])
```
if ok to stay with a list

Reply:
For this strategy: https://github.com/nipy/heudiconv/blob/254dce7f996760dc50d890530a9b028e6851a36c/heudiconv/parser.py#L110
I think that `_UNPACK_FORMATS` needs to remain a `tuple`